rosidl_typesupport_freertps_cpp/cmake/rosidl_typesupport_freertps_cpp_generate_interfaces.cmake

```diff
@@ -148,8 +73,8 @@ set(target_dependencies
   ${rosidl_typesupport_freertps_cpp_GENERATOR_FILES}
   "${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/msg__type_support.hpp.template"
   "${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/msg__type_support.cpp.template"
-  "${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/srv__type_support.cpp.template")
-#  ${_dependency_files})
+  "${rosidl_typesupport_freertps_cpp_TEMPLATE_DIR}/srv__type_support.cpp.template"
+  ${_dependency_files})
```

Comment:
This variable is never populated so can be removed including its initialization in line 50.